code/modules/mob/living/silicon/silicon_mob.dm

```diff
@@ -276,19 +276,19 @@
 
 
 // this function shows the health of the pAI in the Status panel
-/mob/living/silicon/proc/show_system_integrity()
-	if(!src.stat)
-		stat(null, text("System integrity: [round((health/maxHealth)*100)]%"))
-	else
-		stat(null, text("Systems nonfunctional"))
+/mob/living/silicon/proc/show_system_integrity(list/status_tab_data)
+	if(!stat)
+		return list("System integrity:", "[round((health / maxHealth) * 100)]%")
+
```

Comment:
```suggestion

```
same here

Reply:
Thanks, realised I could simplify it further by making a ternary, appreciated for the pointer